### DISCUSSION TITLE
Encountered 500 Fatal Error

### DISCUSSION THREAD

**Discussion by pingfan-hu** [rubenarslan/formr.org#520] [Q&A]
Hi there!

I deployed a formr instance on my Linode server. The page showed up properly and can be accessed here: [172.233.238.100](http://172.233.238.100).
However, I attempted to create an account and got warned an error code of 500, saying:
>500 Fatal Error:
>A fatal error occurred and your request could not be completed. Contact site admins with these details
>Error [1] in application/Session.php line 92

The application/Session.php line 92 writes:
`setcookie(self::REQUEST_TOKENS_COOKIE, $token, 0, self::$path, self::$domain, self::$secure, self::$httponly);`

I wonder if it's a common problem during the deployment. Is it because of cookies or something?

Thanks!

The screenshot is attached here:
<img width="668" alt="image" src="https://github.com/rubenarslan/formr.org/assets/84371500/016b49c2-9c17-47d2-a24f-504bc2db8ec1">

**Comment by rubenarslan** [comment]
Unfortunately, we do not have the time to support custom deploys. We are offering [consulting](https://researchmixtapes.com/) to deploy a dockerised instance of formr. Formr is open source, so you can of course self host. If you decide to self-host and can do it on your own, I am happy to do a check-up that a server is securely set up (as far as I can tell from outside), when you're done. As it stands, this is not secure (no SSL).
